### ISSUE TITLE
Error when trying to build for armv7l

### ISSUE THREAD

**Issue by wencywww** [apache/cordova-electron#132]
# Bug Report

### What is expected to happen?
To build a tar.gz package for linux/arm7

### What does actually happen?
The build process fails with a 404 error when trying to download electron-v4.2.12-linux-arm.zip

### Command or Code

cordova build electron

### Environment, Platform, Device

Raspbian Buster on Raspberry Pi 4 / 4GB (same result under Windows 10)

### Version information

Cordova: 9.0.0 (cordova-lib@9.0.1)
Platforms: electron 1.1.1
OS: Raspbian Buster

Output of cordova build electron:
```
 • electron-builder version=20.44.4
  • writing effective config file=platforms/electron/build/builder-effective-config.yaml
  • installing production dependencies platform=linux arch=armv7l appDir=/home/pi/cordova/myApp/platforms/electron/www
Error: npm exited with code 1
Output:

> ejs@2.7.4 postinstall /home/pi/cordova/myApp/platforms/electron/www/node_modules/ejs
> node ./postinstall.js

Thank you for installing EJS: built with the Jake JavaScript build tool (https://jakejs.com/)


> electron@4.2.12 postinstall /home/pi/cordova/myApp/platforms/electron/www/node_modules/electron
> node install.js

Downloading tmp-3657-0-electron-v4.2.12-linux-arm.zip
Error: GET https://github.com/electron/electron/releases/download/v4.2.12/electron-v4.2.12-linux-arm.zip returned 404

Error output:
/home/pi/cordova/myApp/platforms/electron/www/node_modules/electron/install.js:49
  throw err
  ^

Error: Failed to find Electron v4.2.12 for linux-arm at https://github.com/electron/electron/releases/download/v4.2.12/electron-v4.2.12-linux-arm.zip
    at Request.<anonymous> (/home/pi/cordova/myApp/platforms/electron/www/node_modules/nugget/index.js:169:61)
    at Request.emit (events.js:189:13)
    at Request.onRequestResponse (/home/pi/cordova/myApp/platforms/electron/www/node_modules/request/request.js:1066:10)
    at ClientRequest.emit (events.js:189:13)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:556:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:109:17)
    at TLSSocket.socketOnData (_http_client.js:442:20)
    at TLSSocket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)

```

build.json contains:

```
{
    "electron": {
        "linux": {
            "package": [
                "tar.gz"
            ],
            "arch": [
                "armv7l"
            ]
        }
    }
}
```

## Checklist

- [ x] I searched for existing GitHub issues
- [ x] I updated all Cordova tooling to most recent version
- [ x] I included all the necessary information above

**Comment by wencywww** [comment]
Anybody with a suggestion?

**Comment by timbru31** [comment]
The reason is already included in your log message: `https://github.com/electron/electron/releases/download/v4.2.12/electron-v4.2.12-linux-arm.zip` points to a 404.

Given the fact, that we are preparing the new 2.0 release and have just upgraded Electron from v4 to v7 you could test the nightly build to see if the issue is already resolved in the master:
```
cordova platform rm electron
cordova platform add electron@nightly
cordova build electron
```

**Comment by erisu** [comment]
I can also confirm that I saw the same issue with current released version.

I do not typically build with this type of architecture but when I did some digging, this error has existed before within the electron/builder. This is not a Cordova bug.

As @timbru31 pointed out, we are currently preparing a major release that updated the electron and electron builder utility. When I tested building with master/nightly it the armv7l package properly discovered and build successful.

Currently there is no hard ETA for the next major release but is being worked on.
